packages/react/src/components/Masthead/Masthead.js

```diff
@@ -211,19 +216,27 @@ const Masthead = ({
         );
         const tocPosition =
           tocBoundingClient.top + lastScrollPosition - window.scrollY;
-        tableOfContents.style.top = `${Math.max(
-          Math.min(tocPosition, stickyRef.current.offsetHeight),
-          0
-        )}px`;
 
-        if (tableOfContents.style.top === '0px') {
-          stickyRef.current.style.top = `-${stickyRef.current.offsetHeight}px`;
-        } else if (
-          tableOfContents.style.top === `${stickyRef.current.offsetHeight}px`
+        if (
+          tableOfContents.getRootNode().host.getAttribute('toc-layout') ===
```

Comment:
@IgnacioBecerra this looks to be the line throwing exceptions, it can't seem to find `tableOfContents.getRootNode().host`.